presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -112,9 +112,13 @@ public NativeExecutionProcess(
     /**
      * Starts the external native execution process. The method will be blocked by connecting to the native process's /v1/info endpoint with backoff retries until timeout.
```

Comment:
> with backoff retries until timeout

Curious where this timeout is specified and what is the default value.

Reply:
it's controlled by [MAX_ERROR_DURATION](https://github.com/prestodb/presto/blob/master/presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcessFactory.java#L42)and configurable, default value is 2min.